front/src/layout/AuthLayout.vue

```diff
@@ -88,10 +84,7 @@
           </div>
           <div class="col-xl-6">
             <ul
-              class="
-                nav nav-footer
-                justify-content-center justify-content-xl-end
-              "
+              class="nav nav-footer justify-content-center justify-content-xl-end"
```

Comment:
w innych plikach classy są po enterach, tutaj jest wszystko w jednej linii. przyjmij jeden sposob zapisywania aby ladniej wygladal kod

Reply:
Jeśli długości linii przekracza 120 znaków to `eslint` próbuje jakoś sformatować tag, żeby go skrócić, stąd te entery. Trzymałbym się tego bo raz że to są rekomendowane ustawienia eslinta, a dwa że ma sens bo inaczej byśmy mieli albo bardzo `szerokie` albo bardzo `długie` html-le